ui-speedspacechart/src/components/common/Tooltip.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+type TooltipProps = {
+  cursorX: number;
+  cursorY: number;
+  height: number;
+  text: string;
+};
+const Tooltip = ({ cursorX, cursorY, height, text }: TooltipProps) => {
+  const tooltipWidth = 246;
```

Comment:
Move this const outside of the component, for instance above the type ToolTipProps.

Also, you can use Screaming case, it's a good convention for static const: 
```suggestion
  const TOOLTIP_WIDTH = 246;
```

Reply:
Good idea, done !